### PR TITLE
afl: add reproducing section

### PR DESCRIPTION
**Pull request by cujomalainey**
reproduction with crashes isnt entirely straight forward, give a simple howto on how to inject back into the fuzzer harness without running the full fuzzing rig.

### CODE REVIEW

File: src/afl/tutorial.md [hunk 1]
```diff
@@ -102,3 +102,13 @@ AFL will run indefinitely, so if you want to quit, press `CTRL-C`.
...
+
+## Reproducing
+
+Once you have a few crashes collected from running your fuzzer, you can reproduce them by passing them in manually to your test case. This is typically done via `stdin`. E.g. for `url-fuzz-target` the command would be:
+
+```sh
+cargo afl run url-fuzz-target < out/default/crashes/crash_file
+```
+
+where `out` is the `-o` parameter from your fuzz command and `crash_file` is a arbitrary file in the `crashes` directory.
```
Comment (smoelius) [inline src/afl/tutorial.md:114]:
```suggestion
where `out` is the `-o` parameter from your fuzz command and `crash_file` is an arbitrary file in the `crashes` directory.
```

**Comment by smoelius** [comment]
I have the one small suggestion re "a" vs. "an" but otherwise I think this is fantastic.

**Comment by cujomalainey** [comment]
Updated, thanks for spotting that

**Comment by smoelius** [comment]
Thanks, @cujomalainey. A definite improvement.

**Pull request merged**